DirectivesData.java:37-42: Remove 'line' attribute...

The puzzle `627-fdafed58` from #627 has to be resolved: 

https://github.com/objectionary/jeo-maven-plugin/blob/9b9d6def708c449602362df465ec7f9c08a635bf/src/main/java/org/eolang/jeo/representation/directives/DirectivesData.java#L37-L42

The puzzle was created by rultor on 07-Jul-24. 

Estimate: 90 minutes,  role: DEV. 

If you have any technical questions, don't ask me, submit new tickets instead. The task will be \"done\" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

Suddenly, this issue is still blocked by the following exception:
```java
[ERROR] Failed to execute goal org.eolang:eo-maven-plugin:0.43.2:xmir-to-phi (convert-xmir-to-eo) on project jeo-it: 'PhiMojo' execution failed: java.io.UncheckedIOException: java.io.IOException: java.util.concurrent.ExecutionException: java.lang.IllegalStateException: Shift 'com.yegor256.xsline.StLogged@2cc65825' failed: Shift 'com.yegor256.xsline.StClasspath@7bfe9db5' failed: Failed to transform by net.sf.saxon.jaxp.TransformerImpl: Processing terminated by xsl:message at line 79 in add-refs.xsl; SystemID: file:///org/eolang/parser/add-refs.xsl; Line#: 79; Column#: 40 -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal org.eolang:eo-maven-plugin:0.43.2:xmir-to-phi (convert-xmir-to-eo) on project jeo-it: 'PhiMojo' execution failed
```

See the more here:  https://github.com/objectionary/jeo-maven-plugin/pull/919

--- a/pom.xml
+++ b/pom.xml
@@ -184,7 +184,7 @@ SOFTWARE.
     <dependency>
       <groupId>org.eolang</groupId>
       <artifactId>eo-parser</artifactId>
-      <version>0.43.2</version>
+      <version>0.44.0</version>
     </dependency>
     <dependency>
       <groupId>com.github.volodya-lombrozo</groupId>

--- a/src/it/annotations/pom.xml
+++ b/src/it/annotations/pom.xml
@@ -64,7 +64,7 @@ SOFTWARE.
       <plugin>
         <groupId>org.eolang</groupId>
         <artifactId>eo-maven-plugin</artifactId>
-        <version>0.43.2</version>
+        <version>0.44.0</version>
         <executions>
           <execution>
             <id>convert-xmir-to-eo</id>

--- a/src/it/bytecode-to-eo/pom.xml
+++ b/src/it/bytecode-to-eo/pom.xml
@@ -70,7 +70,7 @@ SOFTWARE.
       <plugin>
         <groupId>org.eolang</groupId>
         <artifactId>eo-maven-plugin</artifactId>
-        <version>0.43.2</version>
+        <version>0.44.0</version>
         <executions>
           <execution>
             <id>convert-xmir-to-phi</id>

--- a/src/it/custom-transformations/pom.xml
+++ b/src/it/custom-transformations/pom.xml
@@ -64,7 +64,7 @@ SOFTWARE.
       <plugin>
         <groupId>org.eolang</groupId>
         <artifactId>eo-maven-plugin</artifactId>
-        <version>0.43.2</version>
+        <version>0.44.0</version>
         <executions>
           <!--
             @todo #610:30min Enable EO Printing in 'custom-transformations' it.

--- a/src/it/exceptions/pom.xml
+++ b/src/it/exceptions/pom.xml
@@ -82,7 +82,7 @@ SOFTWARE.
       <plugin>
         <groupId>org.eolang</groupId>
         <artifactId>eo-maven-plugin</artifactId>
-        <version>0.43.2</version>
+        <version>0.44.0</version>
         <executions>
           <execution>
             <id>convert-xmir-to-eo</id>

--- a/src/it/generics/pom.xml
+++ b/src/it/generics/pom.xml
@@ -64,7 +64,7 @@ SOFTWARE.
       <plugin>
         <groupId>org.eolang</groupId>
         <artifactId>eo-maven-plugin</artifactId>
-        <version>0.43.2</version>
+        <version>0.44.0</version>
         <executions>
           <execution>
             <id>convert-xmir-to-eo</id>

--- a/src/it/jna/pom.xml
+++ b/src/it/jna/pom.xml
@@ -219,7 +219,7 @@ SOFTWARE.
           <plugin>
             <groupId>org.eolang</groupId>
             <artifactId>eo-maven-plugin</artifactId>
-            <version>0.43.2</version>
+            <version>0.44.0</version>
             <executions>
               <execution>
                 <id>xmir-to-phi</id>

--- a/src/it/phi-unphi/pom.xml
+++ b/src/it/phi-unphi/pom.xml
@@ -89,7 +89,7 @@ SOFTWARE.
       <plugin>
         <groupId>org.eolang</groupId>
         <artifactId>eo-maven-plugin</artifactId>
-        <version>0.43.2</version>
+        <version>0.44.0</version>
         <executions>
           <execution>
             <id>xmir-to-phi</id>

--- a/src/it/spring/pom.xml
+++ b/src/it/spring/pom.xml
@@ -84,7 +84,7 @@ SOFTWARE.
       <plugin>
         <groupId>org.eolang</groupId>
         <artifactId>eo-maven-plugin</artifactId>
-        <version>0.43.2</version>
+        <version>0.44.0</version>
         <executions>
           <execution>
             <id>convert-xmir-to-phi</id>

--- a/src/it/takes/pom.xml
+++ b/src/it/takes/pom.xml
@@ -80,7 +80,7 @@ SOFTWARE.
       <plugin>
         <groupId>org.eolang</groupId>
         <artifactId>eo-maven-plugin</artifactId>
-        <version>0.43.2</version>
+        <version>0.44.0</version>
         <executions>
           <execution>
             <id>convert-xmir-to-phi</id>

--- a/src/main/java/org/eolang/jeo/representation/directives/DirectivesEoObject.java
+++ b/src/main/java/org/eolang/jeo/representation/directives/DirectivesEoObject.java
@@ -26,7 +26,6 @@ package org.eolang.jeo.representation.directives;
 import java.util.ArrayList;
 import java.util.Iterator;
 import java.util.List;
-import java.util.Random;
 import org.xembly.Directive;
 import org.xembly.Directives;
 
@@ -78,7 +77,6 @@ public final class DirectivesEoObject implements Iterable<Directive> {
             .attr("base", new EoFqn(this.base).fqn());
         if (!this.name.isEmpty()) {
             directives.attr("name", this.name);
-            directives.attr("line", new Random().nextInt(Integer.MAX_VALUE));
         }
         return directives
             .append(this.inner.stream().reduce(new Directives(), Directives::append))

--- a/src/main/java/org/eolang/jeo/representation/directives/DirectivesJeoObject.java
+++ b/src/main/java/org/eolang/jeo/representation/directives/DirectivesJeoObject.java
@@ -26,7 +26,6 @@ package org.eolang.jeo.representation.directives;
 import java.util.Arrays;
 import java.util.Iterator;
 import java.util.List;
-import java.util.Random;
 import java.util.stream.Collectors;
 import org.xembly.Directive;
 import org.xembly.Directives;
@@ -123,7 +122,6 @@ public final class DirectivesJeoObject implements Iterable<Directive> {
             .attr("base", new JeoFqn(this.base).fqn());
         if (!this.name.isEmpty()) {
             directives.attr("name", this.name);
-            directives.attr("line", new Random().nextInt(Integer.MAX_VALUE));
         }
         return directives
             .append(this.inner.stream().reduce(new Directives(), Directives::append))

--- a/src/main/java/org/eolang/jeo/representation/directives/DirectivesValue.java
+++ b/src/main/java/org/eolang/jeo/representation/directives/DirectivesValue.java
@@ -32,12 +32,6 @@ import org.xembly.Directive;
  * Data Object Directive in EO language.
  *
  * @since 0.1.0
- * @todo #627:90min Remove 'line' attribute usages.
- *  We add 'line' attribute in many places to be able print XMIR representation as PHI expressions.
- *  Actually we shouldn't add any artificial attributes to the representation.
- *  When the following issue will be solved we should remove 'line' attribute from all places
- *  where it used:
- *  https://github.com/objectionary/eo/issues/3189
  */
 @ToString
 public final class DirectivesValue implements Iterable<Directive> {

